docs/pages/routing/navigating-pages.mdx

```diff
@@ -59,12 +59,13 @@ export default function Page() {
 
 ## Imperative navigation
 
-You may want to navigate from a global store when a user logs in or out. You can use the `router` object to navigate imperatively (outside of React).
+You may want to navigate from a global store when a user logs in or out. You can use the `useRouter` hook to navigate imperatively (outside of React).
```

Comment:
```suggestion
You may want to navigate from a global store when a user logs in or out. You can use the `router` object or `useRouter` hook to navigate imperatively (outside of React).
```

The `router` object is a valid statement. If you had trouble using it, I'd suggest opening an issue with a minimal reproducible example.